submission-instructions.md

```diff
@@ -1,9 +1,51 @@
 # Project Submission Instructions
 
 ## Repository Submission Requirements
+
 Each team will be required to submit a GitHub repository with their project. The GitHub repos **must** meet the following criteria:
 
 1. Repository must live within the https://github.com/finos-labs GitHub Org.
-2. Completion of `README.md` details.
-    - Project Name
-    - Team Information
+2. Completion of below information
+
+# 📬 NatWest Hack4ACause hackathon Project Submission Instructions
+
+Please complete this file and include it in your repository (`README.md`) when submitting your hackathon project.
```

Comment:
Perhaps you should put this template into the repositories actual README.md file, move whats in there into a HACK4CAUSE-TEMPLATE_README.md file, and just a line at the top of the new README.md that links over to it for instructions, e.g.

```md
> This repository was forked from [finos-labs/learnaix-h-2025](https://github.com/finos-labs/learnaix-h-2025), please see [ HACK4CAUSE-TEMPLATE_README.md](./HACK4CAUSE-TEMPLATE_README.md) for guidance.
```

you could then merge the content of the submission-instructions.md file into that (instructions currently split between README.mn and submission0instructions.md)